client/my-sites/customer-home/cards/education/affiliates/index.jsx

```diff
@@ -0,0 +1,44 @@
+import { useHasEnTranslation } from '@automattic/i18n-utils';
+import { useTranslate } from 'i18n-calypso';
+import affiliatesIllustration from 'calypso/assets/images/customer-home/illustration--affiliates-icons-small.png';
+import EducationalContent from '../educational-content';
+export const EDUCATION_EARN = 'home-education-earn';
+
+const EducationAffiliates = () => {
+	const translate = useTranslate();
+	const hasTranslation = useHasEnTranslation();
+
+	const title = translate( 'Earn with the Automattic Affiliate Program' );
+	const hasTitleTranaslation = hasTranslation( 'Earn with the Automattic Affiliate Program' );
```

Comment:
Nit: there is a typo
```suggestion
	const hasTitleTranslation = hasTranslation( 'Earn with the Automattic Affiliate Program' );
```